### ISSUE TITLE
AdGuard - Option to configure `ipFamilyPolicy` and `ipFamilies` for services

### ISSUE THREAD

**Issue by hertznsk** [rm3l/helm-charts#86]
Thanks for the AdGuard chart!

By default, Kubernetes services are created to work only on the IPv4 stack.
If, like me, you have a dual stack network, you need to add settings to the services:

```yaml
  ipFamilies:
    - IPv4
    - IPv6
  ipFamilyPolicy: PreferDualStack
```

Could you provide the appropriate service configuration options in the chart?

**Comment by rm3l** [comment]
Hi @hertznsk ,
Thanks for this suggestion, which definitely makes sense. Interested in contributing a PR?

**Comment by hertznsk** [comment]
> Interested in contributing a PR?

Well, I could try to do that. But I'm definitely not good at either k8 or helm.

I'll read more about it and do it next weekend.

**Comment by rm3l** [comment]
I think this has been fixed by #87

Closing but feel free to reopen if you think otherwise.
